tensorly/random/base.py

```diff
@@ -95,7 +95,7 @@ def random_cp(
     if (rank > min(shape)) and orthogonal:
         warnings.warn(
             "Can only construct orthogonal tensors when rank <= min(shape) but got "
-            "a tensor with min(shape)={} < rank={}".format(min(shape), rank)
+            f"a tensor with {min(shape)=} < {rank=}"
```

Comment:
Changed to support Python 3.7
```suggestion
            f"a tensor with min(shape)={min(shape)} < rank={rank}"
```